Client/Frontend/Settings/ClearPrivateDataTableViewController.swift

```diff
@@ -99,18 +105,14 @@ class ClearPrivateDataTableViewController: UITableViewController {
             .map { (_, clearable) in clearable.clear() }
             .allSucceed()
             .upon { result in
-                // TODO: Need some kind of success/failure UI. Bug 1202093.
-                if result.isSuccess {
-                    print("Private data cleared")
-                } else {
-                    print("Error clearing private data")
-                    assertionFailure("\(result.failureValue)")
-                }
+                assert(result.isSuccess, "Private data cleared successfully")
```

Comment:
I think @darrinhenein and @tecgirl were going to look into showing some kind of dialog, but they wanted this minimal UI first. Worth considering now, though, before the string freeze.

FWIW, I think the iOS 8 clear private data hacks will quickly become obsolete as everyone upgrades to 9. And with iOS 9, we shouldn't ever fail since we're just using the built-in `WKWebView` APIs. The `assert` is more of a sanity check that we can catch on dev builds (since `assert`s are not triggered in release builds).